### PR TITLE
Display ratings + unique plays on level page

### PR DESCRIPTION
**Pull request by speedyman09**
image

### CODE REVIEW

File: src/app/pages/level/level.component.html [hunk 1]
```diff
@@ -15,7 +15,12 @@
             updated {{ getMoment(level.updateDate)}}</span-gentle>
         </div>
       </page-header>
-
+      <div class="flex gap-2 pb-3 pl-1">
+        <div class="flex items-center"> <i class="fa-solid fa-thumbs-up pr-1"></i> {{level.yayRatings}} </div>
+        <div class="flex items-center"> <i class="fa-solid fa-thumbs-down pr-1"></i> {{level.booRatings}} </div>
+        <div class="flex items-center"> <i class="fa-solid fa-heart pr-1"></i> {{level.hearts}} </div>
+        <div class="flex items-center"> <i class="fa-solid fa-play pr-1"></i> {{level.uniquePlays}} </div>
```
Comment (jvyden) [inline src/app/pages/level/level.component.html:22]: Don't one-line these, also we use <fa-icon> instead of <i>.

**Review by jvyden — CHANGES_REQUESTED** [review]

File: src/app/pages/level/level.component.html (hunk 1 above)
Comment (speedyman09) [reply src/app/pages/level/level.component.html]: wdym by don’t one line
the div, or everything?

**Comment by jvyden** [comment]
This PR has duplicate code from #25, how do you want me to proceed? Close this PR or the other one?

**Comment by speedyman09** [comment]
Close this PR and keep #25

**Comment by jvyden** [comment]
Sure. I believe the two branches diverge due to review comments so I'll merge things into #25 tomorrow.

**Pull request closed without merging**